code/game/objects/effects/spawners/random_spawners.dm

```diff
@@ -108,3 +112,223 @@
 	result = list(
 	/turf/simulated/wall = 1,
 	/obj/effect/decal/cleanable/fungus = 7)
+
+
+
+// z6 DEPOT SPAWNERS
+
+/obj/effect/spawner/random_spawners/syndicate
+	spawn_nothing_percentage = 50
+
+
+// Turrets
+
+/obj/effect/spawner/random_spawners/syndicate/turret
+	name = "50pc int turret"
+	icon_state = "x"
+	result = list(
+	/obj/machinery/porta_turret/syndicate/interior  = 1
```

Comment:
If you have a list with one thing in it, please don't new line it and put the ending `)` on another new line. It looks bad.